src/Tizen.NET.MaterialComponents/Components/MList.cs

```diff
@@ -0,0 +1,184 @@
+using ElmSharp;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.IO;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public class MList : GenList, IColorSchemeComponent
```

Comment:
As `MList` inherit `GenList`, all public APIs of `GenList` can be exposed. For example, user can use `Append()` to add item. How do you think about this?

Reply:
'MList' thinks it is to provide a 'Material Design', not to constrain its function.
So I think it's a class that makes the 'GenList' look like a 'Material Design', not just a 'Material Design' list.